Data_Structures_Answers.md

```diff
@@ -1,18 +1,25 @@
 Add your answers to the questions below.
 
 1. What is the runtime complexity of your ring buffer's `append` method?
+   O(1)
 
 2. What is the space complexity of your ring buffer's `append` function?
+   O(n)
```

Comment:
Not necessarily. We aren't allocating any additional memory when we appending so it will be O(1) in this case